test/kt/esimd_radix_sort.cpp

```diff
@@ -386,38 +388,53 @@ void test_general_cases(std::size_t size, KernelParam param)
 }
 #endif // TEST_DPCPP_BACKEND_PRESENT
 
+template <typename T, typename KernelParam>
+bool
+can_run_test(KernelParam param)
+{
+    // uint64_t, int64_t, double - skip tests with error: LLVM ERROR: SLM size exceeds target limits
+    if (sizeof(T) == sizeof(::std::uint64_t) && param.data_per_workitem >= 288 && param.workgroup_size == 64)
```

Comment:
Could you investigate how much memory is used by the implementations and compare it against available memory, so the check is more complete? 

`info::device::local_mem_size` device descriptor shows available memory. Our experiments have shown that compiler can implicitly use some SLM, so we can utilize only a part of the device SLM. 
Example: 
https://github.com/oneapi-src/oneDPL/blob/8528820d414921ce04169157e212d5bc6421ba1e/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h#L912

An example is capped to only a half of SLM, I consider it is too strict for ESIMD kernels basing on testing experience. Could you find the most appropriate coefficient?